README.md

```
@@ -106,6 +110,57 @@ Automatic mode using Makefiles, currently supports SNO deployments on two virtua
     - Boot the VM with that ISO.
 - You can now monitor the progress using `abi-wait-complete` or `make ssh` and `journalctl -f -u assisted-service.service` or `kubectl --kubeconfig ./sno-workdir/auth/kubeconfig get clusterversion`.
 
+### Host IP configuration (ABI flow)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)